README.md

```diff
@@ -3,45 +3,30 @@
 Web Application where people new to Machine Learning can input a dataset and experiment around with basic Pytorch modules through a drag and drop interface
 
 > **Deployed website:** https://datasciencegt-dlp.com/ </br>
- **GitHub repo:** https://github.com/karkir0003/Deep-Learning-Playground/ </br> **Owners:** See [CODEOWNERS](./CODEOWNERS)
+ **GitHub repo:** https://github.com/DSGT-DLP/Deep-Learning-Playground </br> 
+ **Owners:** See [CODEOWNERS](./CODEOWNERS)
 
 # How to Run
 
 ## Prerequisites
 Have the following installed first:
 
-1. [NodeJS v16](https://nodejs.org/en/download/) (should come with NPM v8)
+1. [NodeJS v18](https://nodejs.org/en/download/) (should come with NPM v8)
 1. [Anaconda](https://www.anaconda.com/)
 1. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html). After installing, type `aws configure` in your terminal and type in the credentials given in [Secrets](https://docs.google.com/spreadsheets/d/1fRndo-7u0MXghiZoMp3uBepDBW9EghcJ9IL4yS0TdD8/edit?usp=sharing)
 
 ## Recommended
 1. [GitKraken Pro](https://help.gitkraken.com/gitkraken-client/how-to-install/)
 
-## Shortcuts:
-| Terminal   | Procedure   | Command |
-|-----------|-----------|-----|
-| Command Prompt | Run All | `start /B npm run startf & start npm run startb` |
-| Powershell | Run All | `cmd /C 'start /B npm run startf & start npm run startb'` |
-| Linux Terminal | Run All | `tmux new-session -d -s frontend_session 'npm run startf'; tmux new-session -d -s backend_session 'npm run startb'` |
-
-
-## Installing/updating the frontend (one-time)
-In the root directory of this project (`~/Deep-Learning-Playground`), run: 
-```
-npm run installf
-```
-## Installing/updating the backend (one-time)
-```
-npm run installb
-```
-## Running the frontend
-```
-npm run startf
-```
-## Running the backend
-```
-npm run startb
-```
+## To start on localhost:
+| Action                                                   | Command                |
+| -------------------------------------------------------- | ---------------------- |
+| Install / Update Frontend Packages (one-time)            | `npm run installf`     |
+| Install / Update Backend Packages (one-time) | `npm run installb` |
+| Running the Frontend                                     | `npm run startf`       |
+| Running the Backend                         | `npm run startb-mac`   |
```

Comment:
why did this command change to `npm run startb-mac`?

Reply:
That's a mistake. Good catch